lib/config_settings/config.ex

```diff
@@ -127,7 +127,7 @@ defmodule Bonfire.Common.Config do
     get(keys_tree, default, otp_app)
   end
 
-  if Application.compile_env(:bonfire, :env) == :test do
+  if Application.compile_env(:bonfire, :env, :unset) == :test do
```

Comment:
default is `nil` so no need to provide a default value

Reply:
thanks, fixed